### ISSUE TITLE
Decide on license

### ISSUE THREAD

**Issue by darribas** [GDSL-UL/wma#1]
We need to pick a suitable license. The [previous iteration](http://darribas.org/wmn/) was [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) and I'd probably lean towards something like that rather than a software based one like GPL or BSD (this is really an educational material project more than a software one). But I want to open the discussion here to document what we pick and why.

One for @pietrostefani and @darribas to decide.

**Comment by pietrostefani** [comment]
Yes agree - creative commons seems sensible to me. As it's educational material sharing and adaptation both key in my book.
